README.md

```diff
@@ -213,63 +223,69 @@ You may wish to read this Guide all the way through, front to back, or jump to a
 area, depending on your familiarity with the subject.
 
 Equity compensation lies at the intersection of corporate law, taxation, and employee
-compensation, and so requires some basic understanding of all three topics.
+compensation, and so requires some basic understanding of all three.
 You might think compensation and taxation are separate topics, but they are so intertwined
-that it’s hard to explain one without the other.
+it would be misleading to explain one without the other.
 
-So we’ve laid out key elements of these subjects, from the ground up:
+So we’ve laid out the key elements of these subjects, from the ground up:
 
-- [**Fundamentals of Stock Corporations**](#fundamentals-of-stock-corporations): What is equity?
-  To understand the basics, you need to know how companies organize their ownership and the
-  basics of how stock works.
-- [**Equity Compensation Basics**](#equity-compensation-basics): What is equity worth?
-  The next set of fundamentals are around compensation and if and when you can sell equity
-  for cash.
-- [**How Equity is Granted**](#how-equity-is-granted): Equity is granted in many forms, most
-  commonly restricted stock grants, stock options, and RSUs.
-  This covers exactly what these are.
+- [**Fundamentals of Stock Corporations**](#fundamentals-of-stock-corporations): To understand
+  exactly what equity is, it’s essential to know how stock works and how companies at
```

Comment:
Change to: "you need to know how companies organize their ownership, how stock works, and how ownership typically evolves."

This is a nit but again we need to be careful to preserve meaning and not make misleading implications.

The new ending to the sentence now implies the organization itself changes, rather than the ownership percentages.

Reply:
Fixed. (stet "it's essential to know" against "you need to")